camayoc/tests/utils.py

```diff
@@ -1,13 +1,15 @@
 # coding=utf-8
 """Utility functions."""
 
+import functools
 import time
 
 from pyVmomi import vim
 
 from camayoc import cli
 
 
+@functools.lru_cache(maxsize=8)
 def get_vcenter_vms(vcenter_client):
```

Comment:
:eyes: what does this do! (interested)

Reply:
Caching ftw! Worth reading its docs.

Reply:
I've dropped the cache since the client does not stay connect longer and was causing some disconnect issues.